src/Components/Login/Login.tsx

```diff
@@ -65,6 +65,16 @@ export const Login = ({ setLoggedIn }: Props) => {
             <button type="submit" className="login__btn btn btn-primary">
                 Submit
             </button>
+            <button
+                type="submit"
+                className="login__btn btn btn-success"
+                onClick={() => {
+                    setPopulateLocalStorageQuestions(questionBank);
+                    setLoggedIn(true);
+                }}
```

Comment:
I was thinking of putting this as an event handler for the code so it's dry and i don't have to call the same functions in two different places if it's not needed.